app/Domain/Clients/Templates/delClient.blade.php

```diff
@@ -22,8 +22,8 @@
     <div class="maincontentinner">
 
         @displayNotification()
-
-        <h4 class="widget widgettitle">{{ __("subtitles.delete") }}</h4>
+        {{-- {{ __("subtitles.delete") }} --}}
+        <h4 class="widget widgettitle"><i class="fa fa-trash"></i> Delete</h4>
```

Comment:
Why was this changed? This hard codes the english string rather than using the translation. If it's the html being escaped please use `{!! __("subtitles.delete") !!}`